examples/client/lwm2mclient.c

```diff
@@ -416,6 +467,13 @@ static void prv_output_servers(lwm2m_context_t * lwm2mH,
     }
 }
 
+/**
+ * @brief Handles resource value changes or value change reports.
```

Comment:
Value change of what? received,sent message, health status ? It is inconclusive since we are using full context of the lwm2m. What does `handle_value_change` do?